kedro/contrib/io/pyspark/spark_data_set.py

```diff
@@ -235,19 +235,23 @@ def _get_spark():
 
     def _load(self) -> DataFrame:
         load_path = self._fs_prefix + str(self._get_load_path())
+        if load_path.startswith("/dbfs"):
```

Comment:
Since it's already used 3 times, maybe worth moving into a helper function like `_remove_dbfs_prefix`? Similar to `_parse_glob_pattern`, for example.

Reply:
Great suggestion; will do! :)

Reply:
Done! Only took me three weeks. :P